README.md

```diff
@@ -223,6 +223,47 @@ func main() {
 ```
 </details>
 
+<details>
+<summary>Audio Captions</summary>
+
+```go
+package main
```

Comment:
It would be great to add a testable example for this, see https://go.dev/blog/examples and #80

Reply:
I agree, but this isn't trivial for this library where we need to set up a test server to handle these examples.

I've done a few of these before https://hoani.net/guides/software/golang/godocs, and they're pretty cool, but I feel like that should be taken care of in a new PR.

I was also thinking when putting these docs together that it might be time for an `examples` folder, similar to other repos like https://github.com/charmbracelet/bubbletea/tree/master/examples - I'd be happy to put this together as a separate PR since it isn't too much work. What do you think?

Reply:
We can do something akin to `api_test.go`:
```go
apiToken := os.Getenv("OPENAI_TOKEN")
if apiToken == "" {
	t.Skip("Skipping testing against production OpenAI API. Set OPENAI_TOKEN environment variable to enable it.")
}
```

But let's separate this to another PR!